homeassistant/components/plant.py

```diff
@@ -97,8 +105,18 @@ def async_setup(hass, config):
         async_track_state_change(hass, sensor_entity_ids, entity.state_changed)
         entities.append(entity)
 
-    yield from component.async_add_entities(entities)
+        if CONF_GROUP_NAME in plant_config and \
+                plant_config[CONF_GROUP_NAME] is not None:
+            group_name = plant_config[CONF_GROUP_NAME]
+            members = ['{}.{}'.format(DOMAIN, plant_name)]  # own entity id
+            members.extend(sensor_entity_ids)
+            groups.append(group.Group.async_create_group(hass, group_name,
```

Comment:
Look into `helper/entity_component.py` to look how you call the group services:
```
hass.components.group.set_group(....)
```
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/group.py#L147

Reply:
ok, changed it